### ISSUE TITLE
Switch to a free CC license for end-user content

### ISSUE THREAD

**Issue by picsi** [fabd/kanji-koohii#103]
I understand the reason why one would want a noncommercial license, I really do. But it hinders reproduction and modification in unintended ways, as the Wikipedia project noted early on: if they wanted to copy the encyclopedia to a CD-ROM distribute it in poor countries and ask in return for a bit of money to pay for the CD costs, CC-BY-NC would not allow them. 

The actual problem here is profiting over other people's work, which in practice is a problem tackled by CC-BY-SA, since few companies want to be stuck to a copyleft license that allows any competitor to copy their work at any time.

Please consider switching koohii to a free license :)
I know this is not an issue related to the koohii underlying software, but it felt like the right venue to discuss this.
Long live free culture!

**Comment by fabd** [comment]
1. Kanji Koohii is not WIkipedia.
2. I don't see a tangible scenario for distributing kanji mnemonics in "poor countries" on CD-ROMs

Let's talk about tangible use cases here. But really, I'd rather someone who is actually directly concerned by the choice of license for one of their projects to tell me about their actual issue.

Besides nobody ever asked me when they scraped the site and redistribute the stories anyway. So the license is there to encourage people who actually respect other's work to get in touch. In which case, the license isn't a problem. For example, a tangible scenario would be someone in Brazil or Vietnam (countries with lots of interest in Japanese) , who want a site completely in their native language. I can always discuss arrangement for use of stories in any specific scenario.

As an aside, nothing is free. If you invest the time to create and share 2000+ kanji mnemoncis, that's didn't come for free. And they are still your intellectual property. So you are free to reproduce your own creations however you want. And Kanji Koohii makes it easy to export your entire data, stories and flashcards.

**TLDR** I don't see a use case for *not* using CC-BY-NC since I can discuss usage of stories with any interested party in tangible scenarios (say, a mobile app).
